src/watertap_contrib/reflo/costing/units/multi_effect_crystallizer.py

```diff
@@ -0,0 +1,358 @@
+#################################################################################
+# WaterTAP Copyright (c) 2020-2024, The Regents of the University of California,
+# through Lawrence Berkeley National Laboratory, Oak Ridge National Laboratory,
+# National Renewable Energy Laboratory, and National Energy Technology
+# Laboratory (subject to receipt of any required approvals from the U.S. Dept.
+# of Energy). All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. These files are also available online at the URL
+# "https://github.com/watertap-org/watertap/"
+#################################################################################
+
+import pyomo.environ as pyo
+from idaes.core.util.misc import StrEnum
+from idaes.core.util.constants import Constants
+from idaes.core.util.exceptions import ConfigurationError
+from watertap.costing.util import register_costing_parameter_block
+from watertap_contrib.reflo.costing.util import make_capital_cost_var
+
+
+class MultiEffectCrystallizerCostType(StrEnum):
+    mass_basis = "mass_basis"
+    volume_basis = "volume_basis"
+
+
+def build_recovered_nacl_cost_param_block(blk):
+
+    blk.cost = pyo.Param(
+        mutable=True,
+        initialize=0,
+        doc="Recovered cost (sale price) of NaCl",
+        units=pyo.units.USD_2020 / pyo.units.kg,
+    )
+
+    costing = blk.parent_block()
+    costing.register_flow_type("NaCl_recovered", blk.cost)
+
+
+def build_steam_cost_param_block(blk):
+
+    blk.cost = pyo.Param(
```

Comment:
Will this show up on the model as m.fs.costing.unit_name.steam.cost?

Reply:
Will be ```m.fs.costing.steam.cost```